core/src/operator.rs

```diff
@@ -695,6 +696,35 @@ impl Operator {
 
         Ok(txs_to_be_sent)
     }
+
+    /// Generates Winternitz public keys for every watchtower challenge and
+    /// BitVM assert tx.
+    ///
+    /// # Returns
+    ///
+    /// - [`Vec<Vec<winternitz::PublicKey>>`]: Winternitz public keys for
+    ///   `watchtower index` row and `BitVM assert tx index` column.
+    pub fn get_winternitz_public_keys(&self) -> Result<Vec<winternitz::PublicKey>, BridgeError> {
+        let mut winternitz_pubkeys = Vec::new();
+
+        for watchtower in 0..self.config.num_watchtowers as u32 {
```

Comment:
Every operator has time_tx times winternitz pubkeys, no need for watchtower for loop

Reply:
Yes, each operator should have TimeTx amount of winternitz pubkeys for assert txs. Each assert TX has around 2000 bytes of data to sign with winternitz.